src/main/java/reactor/core/publisher/ParallelFlux.java

```diff
@@ -947,6 +947,23 @@ public void subscribe(Consumer<? super T> onNext, Consumer<? super Throwable>
 	}
 
 	/**
+	 * Allows composing operators off the 'rails', as individual {@link GroupedFlux} instances keyed by
```

Comment:
Yep perfect, although I'd also mention/introduce the concept of each rail processing a group in the class-level Javadoc too (if that's not too picky :-) )